forest/src/cli/chain_cmd.rs

```diff
@@ -82,18 +85,32 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
+                let now = OffsetDateTime::now_utc();
+
+                let month = now.month() as u8;
+                let month_string = if month < 10 {
+                    format!("0{}", month)
+                } else {
+                    month.to_string()
+                };
```

Comment:
I believe this entire block can be replaced with `let month = format!("{:02}", now.month());`, unless I'm missing something.

Reply:
Thanks